app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True) ->\
+            None:
```

Comment:
The line continuation character `\` is not necessary here since the line is not too long. You can place the `None:` on the same line as the method signature for better readability.